### PR TITLE
Use gmfMobileMeasure constants

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Review by ger-benjamin — CHANGES_REQUESTED** [review]
Try the mobile desktop: `Cannot read property 'map' of undefined` in the console.
 +, the coordinates have no separator, it is normal ? (It should be `x, y` I guess)

**Comment by sbrunner** [comment]
Thanks @ger-benjamin  it should be fixed :-)

File: contribs/gmf/src/mobile/measure/pointComponent.js [hunk 1]
```diff
@@ -347,8 +287,11 @@ MobileMeasurePointController.prototype.getMeasure_ = function () {
   if (!Array.isArray(center)) {
     throw new Error('Wrong center');
   }
+  if (!this.options.rasterLayers || this.options.rasterLayers.length == 0) {
```
Comment (ger-benjamin) [inline contribs/gmf/src/mobile/measure/pointComponent.js:290]:
```suggestion
  if (!this.options.rasterLayers || this.options.rasterLayers.length === 0) {
```
?

**Review by ger-benjamin — APPROVED** [review]
Looks good now. Still one little comment (then LGTM)

File: contribs/gmf/src/mobile/measure/pointComponent.js (hunk 1 above)
Comment (sbrunner) [reply contribs/gmf/src/mobile/measure/pointComponent.js]: Done ...

**Comment by sbrunner** [comment]
Thanks @ger-benjamin  :-)

**Pull request merged**